code/obj/item/pens_writing_etc.dm

```diff
@@ -39,6 +39,13 @@
 	var/spam_flag_sound = 0
 	var/spam_flag_message = 0 // one message appears for every five times you click the pen if you're just sitting there jamming on it
 	var/spam_timer = 20
+	var/symbol_setting = null
```

Comment:
does this var need to exist? You could just increase the scope of the user's input var instead of taxi'ing it with this one, while also removing an if else block

Reply:
Isn't this var existing the whole point of the PR? The goal seems to be to make it so you set the symbol once and then click to write / draw.